core/src/program.rs

```diff
@@ -93,13 +93,13 @@ impl Program {
     }
 
     /// Get reference to memory pages.
-    pub fn get_allocations(&self) -> &BTreeSet<WasmPageNumber> {
+    pub fn allocations(&self) -> &BTreeSet<WasmPageNumber> {
         &self.allocations
     }
 
-    /// Get mut reference to memory pages.
-    pub fn get_allocations_mut(&mut self) -> &mut BTreeSet<WasmPageNumber> {
-        &mut self.allocations
+    /// Set reference to memory pages.
```

Comment:
For this fn and one above `fn allocations` incorrect doc comment

Reply:
Renamed.